grammar.js

```diff
@@ -411,9 +411,10 @@ module.exports = grammar({
           field('lhs', $.map_side),
           field('rhs', $._expression),
         ),
-        seq(field('lhs', $.map_side), field('rhs', $.map_side)),
+        seq(field('lhs', $.map_side), field('rhs', $._map_rhs)),
       ),
 
+    _map_rhs: ($) => alias(/[^\n\r]+/, $.map_side),
```

Comment:
Looks like this one is needed to be on the external scanner, because mapping like `noremap <leader>y "+y`  are detected like a comment.

Reply:
Yes, I am currently working on making that part of the external scanner, just need more time.